bin/utils/cli-config-utils.js

```diff
@@ -48,6 +48,22 @@ function resolveKnexFilePath() {
     };
   }
 
+  const mjsPath = resolveDefaultKnexfilePath('mjs');
+  if (fs.existsSync(mjsPath)) {
```

Comment:
I would avoid sync method calls in runtime. Since you are dropping Node 8 support anyway, you can use fs.promises instead

Reply:
Agreed/

Reply:
theres a bunch of unrelated code that also uses promisify on the node utils. maybe it makes sense to make this change, along with updating that other code, in a separate pr.?

Reply:
sure